v3/integration/lints/main.go

```diff
@@ -0,0 +1,76 @@
+package main // import "github.com/zmap/zlint/v3/integration/lints"
```

Comment:
what's up with the `// import` comment here?

Reply:
Oh whoops. this is how to `go mod init` the module without having specify the package in the CLI invocation (it would have to be `go mod init github.com/zmap/zlint/v3/integration/lints` in this case).

BUUUUUT we tore away the submodule notion so it should have been deleted

Reply:
```suggestion
package main
```